docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,270 @@ Run Apps on your own cloud (BYOC)
+
+The first step is to create a role with permissions that allow Lightning to make changes to your AWS resources on your behalf.
+
+You can do that in two ways:
+
+* Directly in AWS
+
+* Terraform (recommended for Terraform users)
+
+----
+
+Method 1: Use the AWS Console to create role
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You're going to create a custom role, that uses a custom policy, in AWS using the AWS management console.
+
+These are the settings you'll specify when creating the role:
+
+* **Trusted entity type:** **AWS account**
+
+* **Another AWS account number:** **748115360335**
+
+    .. note::
+
+        This is Lightning AI's account number. It allows Lightning AI to make changes to your AWS resources
+		(like creating EC2 instances for your BYOC cluster) on your behalf.
+
+* **Require external ID:** Provide a secure external ID.
+
+    .. warning::
+
+        AWS does not treat the external ID as a secret. But Lightning AI uses the external ID to access AWS on your behalf. **Do not share the external ID.** To read more on what the AWS external ID is and why it's useful go `here <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html>`_.
+
+* Copy-paste the `contents of this JSON file <https://github.com/Lightning-AI/terraform-aws-lightning-byoc/blob/main/iam_policy.json>`_ into the JSON workspace
+  for the custom policy that the custom role uses.
+
+   .. note:: Ignore the warnings generated by AWS when you create the policy.
+
+If you are an IT person creating this role for someone else (researchers, data scientists, Lightning App creators), assign the role to their AWS account after creating the role.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_setup_shorter.mp4" type="video/mp4" width="100%">
+   </video>
+
+----
+
+Method 2: Use Terraform to set up AWS (advanced)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This process is intended for advanced users only. If you have not used Terraform before, you might want to consider using the AWS console.
+
+#. `Download and install Terraform. <https://www.terraform.io/downloads>`_ If you haven’t used Terraform, `here is their tutorial <https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=terraform_io_download>`_.
+
+#. Install `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html>`_.
+
+#. Using the AWS CLI log into your `AWS account <https://aws.amazon.com/free>`_ using `associated credentials <https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html>`_ that allow you to create resources. For example:
+
+    .. code:: bash
+
+        aws configure sso
+        aws sso login
```

Comment:
This is how we login for our corporate setup. Honestly `aws configure` should be enough without all this extra fluff

Reply:
@nmiculinic What would you like to see? I just want to verify.

Reply:
```
aws configure
```

... done